apps/web-twig-demo/webpack.config.js

```diff
@@ -93,4 +95,10 @@ Encore
     // .enableIntegrityHashes(Encore.isProduction())
 ;
 
-module.exports = Encore.getWebpackConfig();
+const config = Encore.getWebpackConfig();
+
+config.resolve.alias = {
+    '@floating-ui/dom': path.resolve(__dirname, 'node_modules/@floating-ui/dom'),
```

Comment:
I am not sure why the Webpack cannot resolve the path to this module. We should check the web package and try to build and install it before we publish the changes.